src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -260,26 +258,77 @@ export class AccessibleBufferWidget extends DisposableStore {
 		this._editorWidget.focus();
 	}
 
-	private _getContent(startIndex?: number): string {
+	private async _updateModel(): Promise<ITextModel> {
+		if (this._lastMarker?.isDisposed) {
+			this._lines = [];
+		}
+		if (this._xterm.raw.buffer.active.length > this._lines.length) {
+			this._removeViewportContent();
```

Comment:
hm, I guess we should be removing the viewport content regardless as it might have changed?